src/Windowing/Silk.NET.Windowing.Common/Interfaces/IView.cs

```diff
@@ -103,6 +103,11 @@ public interface IView : IViewProperties, IGLContextSource, IVkSurfaceSource, ID
         /// </summary>
         void Reset();
 
+        /// <summary>
+        /// Sets focus to current window.
+        /// </summary>
+        void SetFocus();
```

Comment:
I personally would like this renamed to BringToFront

Reply:
Personally SetFocus is slightly different in my mind than BringToFront. SetFocus means it is the currently active window, BringToFront would just make it the top window, not necessarily the active one.

Reply:
Note that while GLFW calls this glfwFocusWindow, SDL calls this SDL_RaiseWindow which is likely a more descriptive of what it does.

Personally, I do not want a method prefixed with `Set` with no parameters. If we're keeping the focus-focused name, then let's just call it `Focus`.

Reply:
Focus seems fine to me

Reply:
Seems agreeable enough.

Reply:
`Focus` sounds good.

`Raise` sounds like it would could be ambiguous

Reply:
I've renamed the method to `Focus`.